roles/foreman_installer/tasks/upgrade.yml

```
@@ -7,6 +7,10 @@
   import_role:
     name: update_os_packages
 
+- name: 'Upgrade postgresql'
+  import_role:
+    name: theforeman.operations.postgresql_upgrade
```

```suggestion
  import_role:
    name: theforeman.operations.postgresql_upgrade
  when:
      - ansible_os_family == "RedHat"
      - ansible_distribution_major_version == '8'
```

Otherwise it will bomb out on Debian and EL7